### ISSUE TITLE
16. As a user, I want the item detail screen to look pretty

### ISSUE THREAD

**Issue by stacietaylorcima** [the-collab-lab/tcl-7-smart-shopping-list#17]
AC:

* The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

![item-detail](https://user-images.githubusercontent.com/4306/68095390-f5288d80-fe5d-11e9-8e75-579e07562589.png)

**Comment by meganmckissack** [comment]
pushed branch up for styling `DMNT-styling` - branch can cover both issues #17 and #18

**Comment by meganmckissack** [comment]
We're going to test out [Tailwind CSS ](https://tailwindcss.com/)to see if it works for our project

**Comment by nabilkaz** [comment]
Add TailwindCSS to the app. 
Combined resources of:
1. This Smashing Magazine article on [adding TailwindCSS to React](https://www.smashingmagazine.com/2020/02/tailwindcss-react-project/)
2. This [video](https://www.youtube.com/watch?v=FiGmAI5e91M&t=27s) of a person applying the article

**Comment by nabilkaz** [comment]
Alright - This one looks to be in a good place
![image](https://user-images.githubusercontent.com/3664197/85748672-170ca400-b6bd-11ea-9ce2-e451cce6ad9c.png)

If we have time - I'd actually rather pull this information onto the grocery list and delete this page. 
We could add these micro details similar to how Airbnb shares listing details.

![image](https://user-images.githubusercontent.com/3664197/85749034-5a671280-b6bd-11ea-97a5-16eed5bd8448.png)

**Comment by meganmckissack** [comment]
This looks great!  I'm going to work on the other issue.

**Comment by meganmckissack** [comment]
I pushed up most everything I can think of doing for the addItem.js styling but have run into a few issues.

- Tried to add custom colors to the tailwind config by extended colors in the theme but couldn't get it to work, not the end of the wold but I was not feeling super pleased with the standard colors, they're not bad but could be better (I removed the custom colors from the file after it didn't work)

- I couldn't figure out how to change the link svg colors, I was thinking a light grey, light-teal, or white.  I tried background and text color on the image tag in the footer file but couldn't change it.

- I'm open to color changes, I'm unsure of the footer color, tried grays, teals, and purple but like teal the best so far, I also added 'active' for the background color array in the tailwind config file but it doesn't seem to work either.

- the add item button sits behind the nav bar on really small screens, I tried adding a a bottom margin for small or xs and it doesn't seem to do anything `sm:mb-5`. 

- ran into this error `Failed to decode downloaded font: https://fonts.googleapis.com/css2?family=Open+Sans&display=swap
add-item:1 OTS parsing error: invalid version tag` maybe the font path is not connecting?

<br/><br/>

![additem-mobile](https://user-images.githubusercontent.com/2355543/85909956-3bf23b80-b7d1-11ea-920a-de9f22658fe5.png)

<br/><br/>

![additem-full](https://user-images.githubusercontent.com/2355543/85909962-4280b300-b7d1-11ea-9340-41be20b142b3.png)
